src/Microsoft.AspNetCore.Mvc.Core/RequestSizeLimitAttribute.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Microsoft.AspNetCore.Mvc.Core
+{
+    /// <summary>
+    /// A filter that specifies the request body size limit.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+    public class RequestSizeLimitAttribute : Attribute, IResourceFilter
+    {
+        /// <summary>
+        /// Initializes an instance of <see cref="RequestSizeLimitAttribute"/>.
+        /// </summary>
+        /// <param name="bytes">The maximum request body size accepted by the application.</param>
+        /// <param name="disabled">Indicates whether this attribute is disabled.</param>
+        public RequestSizeLimitAttribute(long bytes, bool disabled)
```

Comment:
e.g. [RequestSizeLimit(Bytes=10)], [RequestSizeLimit(Disabled=true)]. Not sure what happens for [RequestSizeLimit] though, I assume it would default to 0, false, which would be weird.